docs/configuration/actions.md

```diff
@@ -268,3 +268,21 @@ actions:
 
 Please note that `%changelog` expects dashes in a changelog entry, but the dash
 would be interpreted by `echo` as an option so we need to quote it.
+
+-----
+If you want to change the working directory of a command or multiple commands, don't do this, it won't work as expected:
+
+```yaml
+actions:
+  post-upstream-clone:
+    - cd /tmp
+    - touch test_file"
+\```
+
+Instead, spawn a shell and change the directory as the first command:
+
+```yaml
+actions:
+  post-upstream-clone:
+    - bash -c "cd ${PACKIT_DOWNSTREAM_REPO}; touch new_file_for_dist_git.tmp"
```

Comment:
I'm thinking if we can make both examples equivalent. It's not really possible to use `cd ${PACKIT_DOWNSTREAM_REPO}` in the first one, because the variable is expanded only in a shell and wrapping `cd` in `bash` is obvious nonsense. Do you think it's fine like this?

Reply:
I am not following you 😅

Reply:
This is nonsense, because `cd` in a subshell only affects that subshell:

```yaml
actions:
  post-upstream-clone:
    - bash -c "cd ${PACKIT_DOWNSTREAM_REPO}"
    - touch new_file_for_dist_git.tmp
```

This is not a good example, because `PACKIT_DOWNSTREAM_REPO` wouldn't be expanded:

```yaml
actions:
  post-upstream-clone:
    - cd ${PACKIT_DOWNSTREAM_REPO}
    - touch new_file_for_dist_git.tmp
```

I just thought it would be nicer to have both examples do the exact same thing, just the first one in a way that doesn't work in practice. But I guess it's fine like it is :slightly_smiling_face:

Reply:
Yes, but ideally this wouldn't be needed and people could just write:

```yaml
actions:
  post-upstream-clone:
    script: |
      cd ${PACKIT_DOWNSTREAM_REPO}
      touch new_file_for_dist_git.tmp
```

We have an open issue for that: https://github.com/packit/packit/issues/1641